alfresco-transformer-base/src/main/resources/application.yaml

```diff
@@ -14,6 +14,9 @@ spring:
   jackson:
     default-property-inclusion: non_empty
 
+activemq:
+  url: ${ACTIVEMQ_URL:false}
```

Comment:
Can we set default values for both ActiveMQ?

Reply:
There is default value (= false) if ACTIVEMQ_URL is not set.

I'm not sure how else we can be backwards compatible *and* support the ability to conditionally configure JMS (rather than http-only) as done via:

@ConditionalOnProperty(name = "activemq.url")